content/applications/sales/sales/send_quotations/orders_and_variants.rst

```diff
@@ -1,78 +1,113 @@
-===================================================
-Use product variants on quotations and sales orders
-===================================================
+===============================================
+Product variants on quotations and sales orders
+===============================================
 
-Product variants can be added to quotations and sales orders using the Product Configurator,
-Variant Grid Entry, or both.
+Product variants can be added to quotations and sales orders using the *Product Configurator*,
+*Variant Grid Entry*, or both.
 
 Configuration
 =============
 
-To enable product variants on quotations and sales orders, go to :menuselection:`Sales -->
-Configuration --> Settings --> Product Catalog` and activate one or both of the
-:guilabel:`Product Configurator` and :guilabel:`Variant Grid Entry` settings. Finally, click
-:guilabel:`Save` to apply the settings.
+To enable product variants on quotations and sales orders, go to :menuselection:`Sales app -->
+Configuration --> Settings`, and under the :guilabel:`Product Catalog` heading, activate one (or
+both) of the :guilabel:`Product Configurator` and :guilabel:`Variant Grid Entry` settings. Finally,
+click :guilabel:`Save` to apply the settings.
 
 .. image:: orders_and_variants/activating-entry-type-settings.png
    :align: center
    :alt: Activating entry type settings.
 
-After saving the settings, configure the product form. Head over to :menuselection:`Sales -->
-Products --> Products` and select any product (or click :guilabel:`Create` to create a new one).
-On the product form, click :guilabel:`Edit`, and then click on the :guilabel:`Variants` tab. Under
-:guilabel:`Sales Variant Selection`, choose either :guilabel:`Product Configurator` or
+After saving the settings, it's time to configure the product form.
+
+To do that, navigate to :menuselection:`Sales --> Products --> Products`, and select the desired
+product to configure (or click :guilabel:`Create` to open a blank product form, and create a new
+one).
+
+On the product form, click :guilabel:`Edit`, and click on the :guilabel:`Variants` tab. Click
+:guilabel:`Add a line` to add an attribute in the :guilabel:`Attribute` column, like `Color` or
+`Size`, for example.
+
+Once the attribute is chosen, select the desired values to apply to that attribute in the
+:guilabel:`Values` column, by clicking into the blank field, and either selecting an option from
+the drop-down menu that appears, or create a new value by typing in the new value, and clicking
+:guilabel:`Create` from the drop-down menu that appears beneath the new value.
```

Comment:
I think breaking this up into at least two sentences would make it a bit easier to follow.

Reply:
Great suggestion! It's definitely too long of a sentence. 👍